src/options.c

```diff
@@ -783,7 +784,7 @@ static void menuKeybinds_update( unsigned int wid, const char *name )
    /* Remove the excess. */
    keybind = keybind_info[selected][0];
    opt_selectedKeybind = keybind;
-   window_modifyText( wid, "txtName", keybind );
+   window_modifyText( wid, "txtName", _(keybind_info[selected][1]) );
```

Comment:
The keybind panel shows a short description like "Target Next" instead of a mnemonic "target_next".